pom.xml

```diff
@@ -95,8 +95,8 @@
     <jenkins.version>2.7.3</jenkins.version>
     <java.level>8</java.level>
     <findbugs.failOnError>false</findbugs.failOnError>
-    <mesos.version>1.0.0</mesos.version>
-    <protobuf.version>2.6.1</protobuf.version>
+    <mesos.version>1.7.0</mesos.version>
```

Comment:
Lets do this in a separate PR if this change doesn't require mesos version or protobuf version to be bumped.